### DISCUSSION TITLE
passing  multiple arguments to -e  option

### DISCUSSION THREAD

**Discussion by anneranch** [lxqt/lxqt#2311] [Q&A]
I need to pass multiple argument  to my object  - "bluetoothctl help" 

This code works as expected 
        params = QStringList()<<"-e"<<"bluetoothctl";

This does not 
        params = QStringList()<<"-e"<<"bluetoothctl" << "-e" << " help";

         processTERMINAL->start(exec,params);

I cannot find in documentation if this is even feasible.

**Comment by yan12125** [comment]
Apparently the same question as https://github.com/lxqt/lxqt/discussions/2306. Let's continue there.
